### DISCUSSION TITLE
Leveling Tracker

### DISCUSSION THREAD

**Discussion by furlyman** [Lailloken/Exile-UI#280] [General]
I had a quick question, I am very new and I was wondering if the new 1.50 version has the more in-depth leveling tips as seen in previous versions? I see that the Exile Leveling site now hides the more detailed tips, is there a way to still have these tips show in the leveling tracker box? Thanks in advance, hopefully this makes sense :)

**Comment by Lailloken** [comment]
v1.50.2 was released today with support for those "hints" as they're now called. They are optional and have to be enabled first: [link](https://github.com/Lailloken/Lailloken-UI/releases/tag/v1.50.2.1-beta)
EDIT: make sure to read the notes carefully if you want to apply the changes to an on-going campaign-run

**Reply by furlyman** [reply]
Fantastic, thank you so much for creating a tool that is so beneficial to new players.

**Comment by PoppSickle** [comment]
Wasnt this more understandable and useful in the past? I remember like simple instructions how to find something like the weavers chambers with like an example picture. I cant find that anymore

Edit: Yeah what I was thinking of was Alira for example, before all you had to do was to press tab and it would visually tell the user how the map is structured and what to look out for. Now you have to read which is less intuitive and can be misunderstood and takes more time. I tried going back to 1.3 but the quest tracker import doesnt work there anymore or Im doing something wrong.

**Reply by Lailloken** [reply]
I had to take the pictures out because (as I was following development of exile-leveling behind the scenes) I knew some form of changes was coming. The pictures were offered based on the wording within the guide (e.g. "find petrified soldiers"), i.e. the script figured out on its own when these pictures were available or not.

The whole exile-leveling guide system was restructured shortly before league-launch, and I wasn't given a heads-up. So I really had to rush things two days before launch to even have support for the new structure (just check yourself how much info is missing if you exclude the optional hints from the ` i ` icons on exile-leveling). There wasn't enough time to also re-implement the old pictures because a lot of the text-queues has moved to these hints and was heavily reworded.

**EDIT: Also, if you have direct feedback with regards to the guide-text or its wording, you should let HeartofPhos know. If these improvements make it into exile-leveling, more people will profit. I just bring these guides into overlay-form. You're basically shooting the messenger.**
